### PR TITLE
E2E tests on run harmonize with flow options

### PR DESCRIPTION
**Pull request by ayuwono**
Adding tests on running harmonize flow

Validating data on browse data page and viewer page after running harmonize flow. The harmonize flow has additional flow options in which the options properties are validated on viewer page.

Validating data on browse data page, viewer page, and dashboard page after running input flow

Some improvement on the code by eliminating browser sleeps

### CODE REVIEW

File: quick-start/e2e/page-objects/flows/flows.ts [hunk 1]
```diff
@@ -202,7 +263,7 @@ export class FlowPage extends AppPage {
     // no progress bar before we start
     // expect(element(by.css('.job-progress')).isPresent()).toBe(false);
 
-    browser.wait(EC.elementToBeClickable(this.mlcpRunButton));
+    //browser.wait(EC.elementToBeClickable(this.mlcpRunButton));
```
Comment (dmcassel) [inline quick-start/e2e/page-objects/flows/flows.ts]: looks like this line can be deleted

File: quick-start/e2e/page-objects/flows/flows.ts [hunk 2]
```diff
@@ -212,8 +273,8 @@ export class FlowPage extends AppPage {
 
     jobsPage.isLoaded();
 
-    browser.wait(EC.presenceOf(jobsPage.firstFinishedFlow));
-    expect(jobsPage.finishedFlows.count()).toBe(count);
+    //browser.wait(EC.presenceOf(jobsPage.firstFinishedFlow));
```
Comment (dmcassel) [inline quick-start/e2e/page-objects/flows/flows.ts]: looks like these two lines can be deleted

Comment (ayuwono) [reply quick-start/e2e/page-objects/flows/flows.ts]: yes, I plan to change the verification step

File: quick-start/e2e/page-objects/flows/flows.ts (hunk 1 above)
Comment (ayuwono) [reply quick-start/e2e/page-objects/flows/flows.ts]: yes, mlcpRunButton (Run Input Flow button) is always presence

**Pull request merged**